plugins/count.yaml

```diff
@@ -0,0 +1,84 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: count
+spec:
+  version: v0.2.5
+  homepage: https://github.com/chenjiandongx/kubectl-count
+  shortDescription: Show resources count in the cluster.
+  description: |
+    kubectl-count uses the dynamic library to found server preferred resources and then
```

Comment:
How you implemented this isn't interesting to users (informers, preferred resources etc). 

Instead, spell out the features you have here. By reading this I can't tell what your plugin does at all.

Reply:
long description has been changed as

> kubectl-count make it easy to count resources by kind. All kinds in your Kubernetes could be retrieved. 
    You can show all resources or show resources in a specified namespace.

Reply:
It has quite a few grammatical errors so I will help you rewrite it if you like:

> This plugin resources on Kubernetes API by kind. It can query all APIs and namespaces and group by them.

Reply:
done. Thanks for your advice